modules/recording/recording/libs/download.ts

```diff
@@ -0,0 +1,98 @@
+import { PDFDocument, rgb, StandardFonts } from "pdf-lib";
+import { saveAs } from "file-saver";
+
+class Download {
+
+  static async downloadAudio(audioUrl: string) {
+    try {
+      const response = await fetch(audioUrl);
+      const blob = await response.blob();
+      const blobUrl = URL.createObjectURL(blob);
+
+      const anchor = document.createElement("a");
+      anchor.href = blobUrl;
+      anchor.download = `audio-${new Date().toLocaleDateString()}.mp3`;
+
+      document.body.appendChild(anchor);
+      anchor.click();
+      document.body.removeChild(anchor);
+
+      URL.revokeObjectURL(blobUrl);
```

Comment:
We have similar function on the code, lets create a util function for it.

Actually I have a `test-util` to test it already

Reply:
I extracted that code from the recording page here

Reply:
I figured we can use this to handle all download functions